tests/output/ob_start_callbacks_bug8866.phpt

```diff
@@ -0,0 +1,72 @@
+--TEST--
+Test ob_start() with callbacks throwing an exception
+--FILE--
+<?php
+
+ob_start(function ($buffer) {
+    return '';
+});
+echo 'never printed - return \'\'';
+ob_end_flush();
+
+ob_start(function ($buffer) {
+    throw new \Exception('x');
+});
+echo 'never printed - throw from callback';
```

Comment:
how to prevent this to be printed in when an exception is raised?

Reply:
the ex should be probably detected here: https://github.com/php/php-src/blob/b9e895bca0a479cfebc63c782f556b016ea07f19/main/output.c#L970